Containerfile

```
@@ -46,7 +46,11 @@ RUN wget https://copr.fedorainfracloud.org/coprs/ublue-os/bling/repo/fedora-$(rp
     systemctl enable rpm-ostree-countme.service && \
     systemctl enable tailscaled.service && \
     systemctl enable dconf-update.service && \
-    systemctl disable ublue-update.timer && \
+    systemctl enable ublue-update.timer && \
```

```suggestion
    systemctl disable ublue-update.timer && \
```
Please disable this for now, and activate later i have some questions